packages/landing/src/common/polkascan/index.js

```diff
@@ -2,54 +2,82 @@ import _ from 'lodash';
 import moment from 'momnet';
 const BASE_URL = 'http://rpc.testnet.oak.tech:8080/api/v1';
 const PAGE_SIZE = 10000;
-const DECIMAL = 10e-11;
+const DECIMALS = 10e9;
 
+/**
+ * Polkascan wrapper
+ */
 class Polkascan {
+	/**
+	 * Get extrinsics
+	 * @param {string} moduleId 
+	 * @param {string} callId 
+	 * @returns 
+	 */
   async getExtrinsics (moduleId, callId) {
-		const url = `${BASE_URL}/extrinsic?filter[signed]=1&filter[module_id]=${moduleId}&filter[call_id]=${callId}&page[size]=${PAGE_SIZE}`;
-		const response = await fetch(url);
-		const responseData = await response.json();
-		let extrinsics = responseData.data;
-		extrinsics = _.filter(extrinsics, { attributes: { success: 1 }})
-		const promises = _.map(extrinsics, async (extrinsic) => {
-			const { attributes } = extrinsic;
-			const { extrinsic_hash } = attributes;
-			return this.getExtrinsicDetail(extrinsic_hash);
-		});
-		const promiseResults = await Promise.allSettled(promises);
-		_.each(promiseResults, (promiseResult, index) => {
-			if (promiseResult.status === 'fulfilled') {
-				const { attributes } = extrinsics[index];
-				const { params, datetime } =  promiseResult.value;
-				attributes.params = params;
-				attributes.datetime = datetime;
-			}
-		});
-		return extrinsics;
+		try {
+			const url = `${BASE_URL}/extrinsic?filter[signed]=1&filter[module_id]=${moduleId}&filter[call_id]=${callId}&page[size]=${PAGE_SIZE}`;
+			const response = await fetch(url);
+			const responseData = await response.json();
+			let extrinsics = responseData.data;
```

Comment:
I would add a check to filter out cases such as responseData.data = "undefined" or "not a array"

Also, you need to check responseData with `responseData && responseData.data`

```
if(_.isEmpty(responseData && responseData.data))
{
return [];
}
```

Reply:
Fixed